cmd/wallet-sdk-gomobile/docs/usage.md

```diff
@@ -1145,7 +1145,8 @@ let opts = Openid4vpNewOpts().setActivityLogger(activityLogger) // Optional, but
 var newInteractionError: NSError?
 let interaction = Openid4vpNewInteraction(args, opts, &newInteractionError)
 let query = interaction.getQuery()
-let inquirer = CredentialNewInquirer(nil)
+var newInquirerError: NSError?
+let inquirer = CredentialNewInquirer(nil,&newInquirerError)
```

Comment:
Formatting: should have a space between the arguments

Reply:
done